ui/src/main/java/com/doordeck/sdk/common/models/DefaultDeviceWithUUID.kt

```diff
@@ -0,0 +1,61 @@
+package com.doordeck.sdk.common.models
+
+import com.doordeck.sdk.common.utils.isUUID
+import com.doordeck.sdk.dto.Role
+import com.doordeck.sdk.dto.device.Device
+import com.doordeck.sdk.dto.device.DeviceSetting
+import com.doordeck.sdk.dto.device.UnlockBetweenWindow
+import java.time.Instant
+import java.util.Optional
+import java.util.UUID
+
+class DefaultDeviceWithUUID(
```

Comment:
What about `PartialDevice` then?

Reply:
top, I'll change it.

> I'm not really sure that's a good argument, everything is a string before its a UUID unless its a brand new UUID!
Well, the easiest and most understandable way to communicate native to multiplatform is through native types such as `String`, `Int`, `Long`, `Char`, `Boolean` and its `[]`s. I can complicate this stuff a bit (indeed my first integration was like that then I changed it.

Reply:
No, we can go with your approach.